kube/services/jobs/google-create-bucket-job.yaml

```diff
@@ -0,0 +1,156 @@
+            GEN3_UNIQUE_BUCKET_NAME
+          - name: STORAGE_CLASS
+            GEN3_STORAGE_CLASS
+          - name: PUBLIC_BUCKET
+            GEN3_PUBLIC_BUCKET
+          - name: REQUESTER_PAYS
+            GEN3_REQUESTER_PAYS
+          - name: GOOGLE_PROJECT_ID
+            GEN3_GOOGLE_PROJECT_ID
+          - name: PROJECT_AUTH_ID
+            GEN3_PROJECT_AUTH_ID
+          - name: ACCESS_LOGS_BUCKET
+            GEN3_ACCESS_LOGS_BUCKET
+        volumeMounts:
+          - name: "creds-volume"
+            readOnly: true
+            mountPath: "/var/www/fence/creds.json"
+            subPath: creds.json
+          - name: "config-volume"
+            readOnly: true
+            mountPath: "/var/www/fence/local_settings.py"
+            subPath: local_settings.py
+          - name: "json-secret-volume"
+            readOnly: true
+            mountPath: "/var/www/fence/fence_credentials.json"
+            subPath: fence_credentials.json
+          - name: "fence-google-app-creds-secret-volume"
+            readOnly: true
+            mountPath: "/var/www/fence/fence_google_app_creds_secret.json"
+            subPath: fence_google_app_creds_secret.json
+          - name: "fence-google-storage-creds-secret-volume"
+            readOnly: true
+            mountPath: "/var/www/fence/fence_google_storage_creds_secret.json"
+            subPath: fence_google_storage_creds_secret.json
+          - name: "fence-yaml"
+            mountPath: "/var/www/fence/user.yaml"
+            subPath: user.yaml
+          - name: "fence-jwt-keys"
+            readOnly: true
+            mountPath: "/fence/keys"
+          - name: "config-helper"
+            readOnly: true
+            mountPath: "/var/www/fence/config_helper.py"
+            subPath: config_helper.py
+        command: ["/bin/bash"]
+        args:
+          - "-c"
+          - |
+            if [[ -z "$UNIQUE_BUCKET_NAME" ]]; then
+                echo Cannot create bucket. UNIQUE_BUCKET_NAME variable must be specified.
+                exit 1 # terminate and indicate error
+            fi
+
+            create_command="google-bucket-create --unique-name $UNIQUE_BUCKET_NAME"
+
+            if [[ -n "$STORAGE_CLASS" ]]; then
+                create_command+=" --storage-class $STORAGE_CLASS"
+            fi
+            if [[ -n "$PUBLIC_BUCKET" ]]; then
+                create_command+=" --public"
+            fi
+            if [[ -n "$REQUESTER_PAYS" ]]; then
+                create_command+=" --requester-pays"
+            fi
+            if [[ -n "$GOOGLE_PROJECT_ID" ]]; then
+                create_command+=" --google-project-id $GOOGLE_PROJECT_ID"
+            fi
+            if [[ -n "$PROJECT_AUTH_ID" ]]; then
+                create_command+=" --project-auth-id $PROJECT_AUTH_ID"
+            fi
+            if [[ -n "$ACCESS_LOGS_BUCKET" ]]; then
+                create_command+=" --access-logs-bucket $ACCESS_LOGS_BUCKET"
+            fi
+
+            fence-create ${create_command}
```

Comment:
Add another block here:
```
if [[ $? != 0 ]]; then
  echo "WARNING: non zero exit code: $?"
fi
```
So the script always succeeds (always has exit code 0) - just logs failures, and we avoid Kubernetes annoying retry logic kicking in ...